blocks/transactions/transactions.css

```diff
@@ -0,0 +1,152 @@
+.transactions table {
```

Comment:
All css definitions for a block should be qualified by the block name and the keyword 'block' e.g. `.transactions.block`